apps/main/src/llamalend/PageLlamaMarkets/header-tooltips/BorrowRateHeaderTooltipContent.tsx

```diff
@@ -0,0 +1,27 @@
+import { t, Trans } from '@ui-kit/lib/i18n'
+import { TooltipDescription, TooltipWrapper } from '@ui-kit/shared/ui/TooltipComponents'
+
+export const BorrowRateHeaderTooltipContent = () => (
+  <TooltipWrapper>
+    <TooltipDescription
+      text={t`The borrow rate is the cost related to your borrow. Depending on the market type it is correlated to different variables.`}
+    />
+    <TooltipDescription
+      text={
+        <Trans>
+          For <strong>lending markets</strong> it varies according to the market utilization.{' '}
```

Comment:
fyi you can use `Typography component="em"` for example - however I just read that mui properly renders `em` and `strong` elements inside `Typography` so this is ok